spec/post_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  it { should validate_presence_of(:title) }
+  it { should validate_length_of(:title).is_at_most(250) }
+  it { should validate_numericality_of(:comments_counter).only_integer.is_greater_than_or_equal_to(0) }
+  it { should validate_numericality_of(:likes_counter).only_integer.is_greater_than_or_equal_to(0) }
+
+  describe '#update_post_counter' do
+    it 'updates the posts_counter of the associated user' do
+      user = create(:user) # Assuming you have a factory for User
+      post = create(:post, user:) # Assuming you have a factory for Post
+
+      expect { post.update_post_counter }.to change { user.reload.posts_counter }.by(1)
+    end
+  end
+
+  describe '#five_recent_comments' do
+    it 'returns the five most recent comments for the post' do
+      post = create(:post) # Assuming you have a factory for Post
+      comments = create_list(:comment, 6, post:) # Assuming you have a factory for Comment
+
+      five_recent_comments = post.five_recent_comments
+
+      expect(five_recent_comments.length).to eq(5)
+      expect(five_recent_comments).to include(comments[0], comments[1], comments[2], comments[3], comments[4])
+      expect(five_recent_comments).not_to include(comments[5])
+    end
+  end
+end
```

Comment:
### ALL TEST FAILS IN ALL TEST FILES

- Hey, please, kindly re-evaluate your test, they are all failing. Please, kindly fix them in all the tests files, i.e, 
`comment_spec, user_spec, post_spec and like_spec`

*failed post spec. I only added image for post spec, kindly resolve ALL*
![image](https://github.com/Nessrine88/Blog-App/assets/59397159/bd8a9125-82c3-4932-82d1-acb135dcb31b)